pkg/rpc/adminservice/version.go

```diff
@@ -0,0 +1,16 @@
+package adminservice
+
+import (
+	"context"
+
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+func (m *AdminService) GetVersion(ctx context.Context, versionRequest *admin.GetVersionRequest) (*admin.GetVersionResponse, error) {
+
+	response, err := m.VersionManager.GetVersion(ctx, versionRequest)
```

Comment:
can we add a catch for panics like here: https://github.com/flyteorg/flyteadmin/blob/master/pkg/rpc/adminservice/project.go#L17?